components/utilities/dialog/index.jsx

```diff
@@ -296,7 +296,6 @@ class Dialog extends React.Component {
 	};
 
 	handleClose = (event, data) => {
-		this.setState({ triggerPopperJS: true });
```

Comment:
This is used. I don't know why it's being removed.

Reply:
@interactivellama I removed it because the function handleClose is being called during componentWillUnmount() and updating the state has no effect here. I have added it again in the code.

Reply:
Good catch! You are correct. Please revert.

Reply:
I'll go ahead and remove it, since it was my bad call in the first place.

Reply:
I reviewed the code again. `handleClose` is also called on handleClickOutside and that's why it should be present. I'm going to go ahead and merge this.

```
handleClickOutside = () => {
		this.handleClose();
	};
```